### PR TITLE
Feature/gurux update

### PR DESCRIPTION
**Pull request by raymar9**
- set and freeze gurux-dlms to 1.0.107

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,11 +51,11 @@
     install_requires=[
         "aioserial==1.3.0",
         "asyncio-mqtt==0.10.0",
-        "gurux-dlms==1.0.104",
+        "gurux-dlms==1.0.107",
         "paho-mqtt==1.5.1",
         "pyserial==3.5",
     ],
-    scripts=['bin/smartmeter-datacollector'],
+    scripts=["bin/smartmeter-datacollector"],
```
Comment (Montellese) [inline setup.py:58]: This is unrelated. Please move it to a separate commit.

File: smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py [hunk 2]
```diff
@@ -21,17 +21,20 @@
 
 
 class HdlcDlmsParser:
-    HDLC_BUFFER_MAX_SIZE = 10000
+    HDLC_BUFFER_MAX_SIZE = 5000
```
Comment (Montellese) [inline smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py:24]: Is there any reason for reducing this?

**Review by Montellese — CHANGES_REQUESTED** [review]
Looks good apart from the minor.

File: smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py (hunk 2 above)
Comment (raymar9) [reply smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py:24]: Yes, I wanted to have the capacity of about 10 times the default L+G meter message, but first missed that 10000 hex characters are indeed only 5000 bytes...
In a separate commit now.

File: setup.py (hunk 1 above)
Comment (raymar9) [reply setup.py:58]: done

**Review by Montellese — APPROVED** [review]
